doc/source/user/c-info.ufunc-tutorial.rst

```diff
@@ -18,7 +18,7 @@ Before reading this, it may help to familiarize yourself with the basics
 of C extensions for Python by reading/skimming the tutorials in Section 1
 of `Extending and Embedding the Python Interpreter
 <http://docs.python.org/extending/index.html>`_ and in `How to extend
-NumPy <http://docs.scipy.org/doc/numpy/user/c-info.how-to-extend.html>`_
+NumPy <c-info.how-to-extend.html>`_
```

Comment:
Guessing this `.html` can be stripped too?

Is there any easy way to check the rendered docs for PRs like this?

Reply:
Fixed and squashed, good catch. The html generates a valid http link in the document, so I don't know how we could catch it other than a grep,

`grep "html>" -r . --include='*.rst' --include=*.py | grep -v http`

now catches only the internal links to NEP docs, which are correct since they point to documents not rendered via the numpy repe